File: src/classes/api/req_res/update-lantana-response.ts
'use strict'

import { Lantana } from '@/classes/datas/lantana'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateLantanaResponse extends GkillAPIResponse {

    updated_lantana: Lantana

    updated_kyou: Kyou | null

    constructor() {
        super()
        this.updated_lantana = new Lantana()
        this.updated_kyou = null
    }

}

File: src/server/gkill/api/req_res/update_lantana_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateLantanaResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedLantana *reps.Lantana `json:"updated_lantana"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}
